SETUP.md

```diff
@@ -20,5 +20,11 @@ if you wish.
 Make sure to read the [Crates and Modules](https://doc.rust-lang.org/stable/book/crates-and-modules.html) chapter if you
 haven't already, it will help you with organizing your files.
 
+## Feedback, Issues, Pull Requests
+
+The [exercism/xrust](https://github.com/exercism/xrust) repository on GitHub is the home for all of the Rust exercises. If you have feedback about an exercise, or want to help implement new exercises, head over there and create an issue. Members of the [rust track team](https://github.com/orgs/exercism/teams/rust) are happy to help!
```

Comment:
rust team *might* only visible to those who are already members of the org. will check with a logged out win shortly